libs/explorers/util/src/lib/wiki/wiki.component.ts

```diff
@@ -22,42 +13,42 @@ import { finalize } from 'rxjs/operators';
   styleUrls: ['./wiki.component.scss'],
   encapsulation: ViewEncapsulation.None,
 })
-export class WikiComponent implements OnChanges, OnInit {
+export class WikiComponent implements OnInit {
   synapseApiService = inject(SynapseApiService);
   domSanitizer = inject(DomSanitizer);
   private destroyRef = inject(DestroyRef);
 
-  @Input() ownerId: string | undefined;
-  @Input() wikiId: string | undefined;
-  @Input() className = '';
+  wikiParams = input<SynapseWikiParams>();
 
+  className = '';
   isLoading = true;
-
-  data: SynapseWiki = {} as SynapseWiki;
+  data: SynapseWikiMarkdown = {} as SynapseWikiMarkdown;
   safeHtml: SafeHtml | null = '<div class="wiki-no-data">No data found...</div>';
 
-  ngOnChanges(changes: SimpleChanges): void {
-    if (changes['wikiId'] && !changes['wikiId'].firstChange) {
-      this.getWikiData();
-    }
+  ngOnInit() {
+    this.getWikiMarkdown();
   }
 
-  ngOnInit() {
+  getWikiMarkdown() {
+    const ownerId = this.wikiParams()?.ownerId;
+    const wikiId = this.wikiParams()?.wikiId;
+    if (!ownerId || !wikiId) {
+      console.error('Wiki parameter(s) missing');
+      return;
+    }
+
     this.isLoading = true;
-    this.getWikiData();
-  }
 
-  getWikiData() {
     this.synapseApiService
-      .getWiki(this.ownerId || 'syn25913473', this.wikiId || '')
```

Comment:
removing default value and empty string alternatives in favor of checking for valid values above.